carla-carma-integration/carla_carma_bridge/launch/carla_carma_bridge.launch

```diff
@@ -20,28 +20,37 @@
 -->
 
 <launch>
+  <param name="use_sim_time" value="True"/>
   <arg name='host' default='localhost'/>
   <arg name='port' default='2000'/>
   <arg name='town' default='Town02'/>
   <arg name='vehicle_filter' default='vehicle.toyota.prius'/>
   <arg name='role_name' default='ego_vehicle'/>
   <arg name='spawn_point' default=''/>
 
-  <!-- carla autoware bridge -->
+  <arg name='speed_Kp' default='0.05'/>
```

Comment:
Are these PID params separate from the CARMA controller ones? Should they be the same?

Reply:
The PID default value has been set as the same value from CARMA [platooning_control config](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/platooning_control/config/parameters.yaml)